content/authorization/what-do-you-get/accesspackages/citizens/familieFritid/_index.en.md

```
@@ -11,31 +11,26 @@ aliases:
 |**Tilgangspakker i kategorien**|
 |---|
 |[Barn og foreldre](#barn-og-foreldre)|
-|[Fullmakter](#fullmakter)|
 |[Barnehage, SFO og skole](#barnehage-sfo-og-skole)|
 |[Samliv](#samliv)|
 |[Fritidsaktiviteter og friluftsliv](#fritidsaktiviteter-og-friluftsliv)|
 
 
 ### Barn og foreldre
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)